new/LIT6955Lamangesta.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6955Lamangesta" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለመንግሥተ፡ ሰማይ፡ ገነት፡ ዘመዓድም፡ ጽጌሃ፤</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Lamangǝśta samāy gannat zamaʿāddǝm ṣǝgehā,...  (Qǝne of ʿǝṭāna mogar type)</title>
+                <title xml:lang="en" corresp="#t1">For the Kingdom of Heaven, Paradise, of its flowers being pleasant,...</title>
```

Comment:
I think the capitalisation is inconsistent

Reply:
I wrote "paradise".